docs/_i18n/en/getting-started/grunt.md

```diff
@@ -10,29 +10,20 @@ To install Grunt, you must **first** [download and install node.js](http://nodej
 
 Then, from the command line:
 
-1. Install `grunt-cli` globally with `npm install -g grunt-cli`.
-2. Navigate to the root `/bootstrap-table/` directory, then run `npm install`. npm will look at the `package.json` file and automatically install the necessary local dependencies listed there.
+1. Navigate to the root `/bootstrap-table/` directory, then run `npm install`. npm will look at the `package.json` file and automatically install the necessary local dependencies listed there.
 
 When completed, you'll be able to run the various Grunt commands provided from the command line.
 
 ## Available Grunt commands
 
-### `grunt dist` (Just compile CSS and JavaScript)
-
-Regenerates the `/dist/` directory with compiled and minified CSS and JavaScript files. As a Bootstrap user, this is normally the command you want.
-
-### `grunt test` (Run tests)
```

Comment:
I removed this because when trying to run it locally, I got the following:

```
> grunt test
Warning: Task "test" not found. Use --force to continue.

Aborted due to warnings.
```